app/src/app/api/v0/user/invites/route.ts

```diff
@@ -123,8 +123,13 @@ export const POST = apiHandler(async (req, { params, session }) => {
             }
           }),
         );
+        // Check if invited user exists then flag that user if they don't exist in the database
+
+        const doesInvitedUserExist = await db.models.User.findOne({
+          where: { email },
+        });
         const host = process.env.HOST ?? "http://localhost:3000";
-        const url = `${host}/user/invites?cityIds=${encodeURIComponent(invites.map((i) => i.cityId).join(","))}&token=${encodeURIComponent(invitationCode)}&email=${encodeURIComponent(email)}`;
+        const url = `${host}/user/invites?cityIds=${encodeURIComponent(invites.map((i) => i.cityId).join(","))}&token=${encodeURIComponent(invitationCode)}&email=${encodeURIComponent(email)}&doesInvitedUserExist=${doesInvitedUserExist ? "true" : "false"}`;
```

Comment:
Might make sense to use URLSearchParams here too to construct the search string rather than doing it manually?